src/plots/pad_attributes.js

```diff
@@ -8,29 +8,35 @@
 
 'use strict';
 
+// This is used exclusively by components inside component arrays,
```

Comment:
Thanks for the note. We might have to make `pad_attributes.js` export a function similar to `font_attributes.js` is ever we want to add `pad` to e.g. `legend` and `colorbar`.